di-ipv-credential-issuer-stub/src/main/java/uk/gov/di/ipv/stub/cred/service/ConfigService.java

```diff
@@ -21,11 +22,14 @@ public class ConfigService {
     private static final Gson GSON = new Gson();
     private static final SsmClient SSM_CLIENT = getSsmClient();
     private static final String CLIENT_CONFIG_BASE_PATH = "/stubs/credential-issuer-stub-clients";
+    private static final String API_KEY_PATH = "/stubs/credential-issuer-stub-api-key";
     private static final String ENVIRONMENT_ENV_VAR = "ENVIRONMENT";
     private static final String TEST = "TEST";
     private static final String CONFIG_CACHE_SECONDS_ENV_VAR = "CONFIG_CACHE_SECONDS";
     private static Map<String, ClientConfig> CLIENT_CONFIGS;
-    private static Instant lastRefresh = null;
+    private static Instant lastClientConfigRefresh = null;
+    private static String apiKey = null;
+    private static Instant lastApiKeyRefresh = null;
```

Comment:
This isn't particularly scalable, but maybe worth saving for a more comprehensive config overhaul. We can probably just use an out-of-the-box caching library if there isn't an equivalent of powertools.

That could also have useful features like stale-while-revalidate etc.